### DISCUSSION TITLE
Collapse - Content resizing within a Collapse-Component

### DISCUSSION THREAD

**Discussion by SergejGleithmann** [mdbootstrap/TW-Elements-React#171] [Share ideas & request features]
Hello, I observed, that the Collapse-Component makes use of style attributes instead of using Tailwind classes for showing and hiding the Collapse-Component. Maybe I am missing the point, but imo this is bad, because the height is set to an absolute px value and therefore does not adjust if content within the collapse comonent changes size. I think it would be better, if the height is not set via the style tag or at least omit the positiv height in case of an opened Collape-Component.

**Comment by iprzybysz** [comment]
Hi @SergejGleithmann, thanks for reporting this issue. We have included it in our task list.

**Reply by SergejGleithmann** [reply]
Thank you for doing so. Is there currently any development on this package?

**Reply by iprzybysz** [reply]
When it comes strictly to TWE React - it is still in Alpha, and the development is frequently interrupted as of now.
At this stage we're focusing mainly on gathering & analyzing feedback and ideas from users that tried to utilize this package in their project.
